lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -2150,28 +3233,33 @@ class DomIntl {}
 
 extension DomIntlExtension on DomIntl {
   // ignore: non_constant_identifier_names
-  external Object? get Segmenter;
+  external JSAny? get Segmenter;
 
   /// This is a V8-only API for segmenting text.
   ///
   /// See: https://code.google.com/archive/p/v8-i18n/wikis/BreakIterator.wiki
-  external Object? get v8BreakIterator;
+  external JSAny? get v8BreakIterator;
 }
 
 /// Similar to [js_util.callMethod] but allows for providing a non-string method
 /// name.
-T _jsCallMethod<T>(Object o, Object method, [List<Object?> args = const <Object?>[]]) {
+T _jsCallMethod<T>(Object o, Object method,
```

Comment:
Should this be hoisted to `js_utils`?

Reply:
Yes, just have to land https://dart-review.googlesource.com/c/sdk/+/288641. Hopefully sometime early next week.